docs/_running-tests/chrome_android.md

```diff
@@ -19,7 +19,7 @@ Until we find a better way, we need to root the Android device and update the
 127.0.0.1   www2.web-platform.test
 127.0.0.1   xn--n8j6ds53lwwkrqhv28a.web-platform.test
 127.0.0.1   xn--lve-6lad.web-platform.test
-0.0.0.0     nonexistent-origin.web-platform.test
+0.0.0.0     nonexistent.web-platform.test
```

Comment:
Presumably as with all other Linux systems, this should be dropped.

Reply:
I haven't verified any of this from an Android system, so I don't feel comfortable changing the instructions. But if I really want to resolve this issue, then that is my responsibility. I'll work on running WPT there, and I'll let you know what I find.